web/portal/client/src/entities/Country/Country.tsx

```diff
@@ -18,10 +18,7 @@ const country: EntityInterface = {
   iden: 'Country',
   title: _('Country', { count: 2 }),
   path: '/countries',
-  toStr: (row: any) => {
-    const language = getI18n().language.substring(0, 2);
-    return row.name[language];
-  },
+  toStr: (row: CountryPropertyList<EntityValue>) => row.countryCode as string,
```

Comment:
Country must show country code or name depending on the entity is using it. You'll have to create another SelectOptions to handle this